parameter/ParameterMgr.cpp

```diff
@@ -739,7 +739,7 @@ void CParameterMgr::applyConfigurations()
     LOG_CONTEXT("Configuration application request");
 
     // Lock state
-    CAutoLock autoLock(getBlackboardMutex());
+    std::lock_guard<std::mutex> autoLock(getBlackboardMutex());
```

Comment:
There is not much repetition here, the `using` trick would not make us gain any line.